JavaErickHernandez/Day3/HashSets.java

```diff
@@ -0,0 +1,51 @@
+package bootcamp;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.HashSet;
+
+public class HashSets {
+	final static String numberOfInputs = "Enter the number of pairs: ";
+	final static String tryAgainMessage = "Try Again";
+	final static String enterFirstName = "Enter first name: ";
+	final static String enterSecondName = "Enter second: ";
+	final static String error = "Error!";
+	final static HashSet<String> nameList = new HashSet<String>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices